### PR TITLE
WIP | PR | The docker image has been created and the docker file has been attached

### PR DESCRIPTION
**Pull request by iScous**
Work In Progress: Follow documentation in README.md;
Issue: #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
...
+
+  It was very helpful for me to use the article about how to create docker file, docker image and push it to the docker hub from this link. [https://buddy.works/guides/how-dockerize-node-application]
+
+  Steps that i followed to complete this issue are:
+
+  * Installed docker from the offical site
+  * Created a docker file with the correct instructions. It includes a package.json file with all the dependencies needed to install correctly node.js, npm and frameworks
+  * Created a docker image from the docker file
+  * Builded the docker image
+  * Pushed the docker image to the docker hub
```
Comment (arsenykonohovwork) [inline README.md]: @iScous You have to add examples of commands that we have to use for creating, deploy, run container;

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM node:8.12.0
+RUN mkdir /src
+WORKDIR /src
+COPY ./package.json /src/package.json
+COPY ./package-lock.json /src/package-lock.json
+COPY ./index.js /src/index.js
+RUN npm install
+COPY . /app
+CMD node /src/index.js
```
Comment (arsenykonohovwork) [inline Dockerfile]: @iScous You have to fix STDOUT output, cause we run process in container and can not walk through it;

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,10 @@
+FROM node:8.12.0
+RUN mkdir /src
+WORKDIR /src
+COPY ./package.json /src/package.json
```
Comment (arsenykonohovwork) [inline Dockerfile]: @iScous where is package.json in source code?

File: Dockerfile (hunk 2 above)
Comment (iScous) [reply Dockerfile]: @arsenykonohovwork  so the src folder cannot be used?

File: README.md (hunk 1 above)
Comment (iScous) [reply README.md]: @arsenykonohovwork added the commands to the file.

File: Dockerfile (hunk 3 above)
Comment (iScous) [reply Dockerfile]: @arsenykonohovwork you mean it must be attached?

**Comment by iScous** [comment]
@arsenykonohovwork ready for review. Added package.json, enabled to navigate in the running container. And for this PR as README.md was added previously i remained it here and modified

**Comment by iScous** [comment]
@arsenykonohovwork the docker-compose.yml was added

File: README.md [hunk 4]
```diff
@@ -1,2 +1,43 @@
...
+```
+To enable to navigate in docker container while it's running the package.json was modified with the following string in scripts section:
+```sh
+"start": "http-server ./app -p 8080 -c-1"
+```
+
+To specify multiple containers we can use docker-compose.yml file. To create and start containers it's possible to use the command:
+```sh
+docker-compose up -d
+```
```
[the remark this answers is not in the document]
Comment (iScous) [reply README.md]: I think yes. I'll try to do that

**Comment by iScous** [comment]
@arsenykonohovwork made changes with relation to your latest comments

**Comment by arsenykonohovwork** [comment]
@iScous this PR is ready for merge;

**Comment by arsenykonohovwork** [comment]
But work still in progress;
So you (@iScous) shouldn't close related issue;

**Pull request merged**